### DISCUSSION TITLE
[BUG] Not found Plots, after switching the SSD

### DISCUSSION THREAD

**Discussion by Parkey18** [Chia-Network/chia-blockchain#3264] [Support]
For a better mining process I changed my M.2 SSD 1 TB to a 2TB. And had to reinstall windows. After installing Chia again, and it fully synced, my **plots couldn`t be found** after setting the right directory.

All I see under the tab "Plots" in the GUI is "**Not found plots. Caution, deleting these plots will delete them forever. Check that the storage devices are properly connected.**"

There are 29 Plots I can`t use right now. I worked for days to get 29 and now I am crushed. 

**What I tryed:**
-> install chia.exe again
-> install an older version of Chia.exe
-> deleting the .chia folder 
-> Crying and rolling on the floor

I used Windows 10, and the GUI.

![Unbenannt](https://user-images.githubusercontent.com/83296039/116267285-91460300-a77c-11eb-8daa-b34b2a2fc306.jpg)

**Comment by lvcivs** [comment]
Have you checked the file `config.yaml` ? Check for `plot_directories` in that file, you might have an old path in there.

**Comment by Parkey18** [comment]
Thank you for the tip, I found it. 
But there is the same directory, where the plots are. 
Any other ideas?

**Comment by Crypto-78** [comment]
Follow... I have the same problem 😭

**Comment by lvcivs** [comment]
Is it possible that you are on a different set of keys after reinstalling? The plots are tied to your keys.

**Comment by Parkey18** [comment]
> Is it possible that you are on a different set of keys after reinstalling? The plots are tied to your keys.

The key is the same. I always had one and registered it with this 20 words. So it should be the same.

**Comment by Crypto-78** [comment]
I have installed the software again, restarted the PC and now everyone can see it

**Comment by Parkey18** [comment]
Mhh, doesnt worked for me :(

**Comment by marlo-cale** [comment]
a mi me paso lo mismo no lo puedo resolver alguien se lo ocurre que puede ser encima sigo haciendo parcelas y no la encuentras encuentro ni  a las nuevas

**Comment by FoodIsGood** [comment]
same problem on ubuntu

**Comment by FoodIsGood** [comment]
I  found a solution. Delete completely config.yaml file. Then use init procedure

**Reply by Bone117** [reply]
And no effect

**Comment by Masty88** [comment]
same problem after creating plot the network can find the file config.yaml is ok,
anyone have suggestion

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
